.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  build:
+    name: "${{ matrix.tests-type }} tests: py${{ matrix.python-version }} on ${{ matrix.os }}"
+    strategy:
+      # run all tests even if e.g. image tests fail early
+      fail-fast: false
+      matrix:
+        os:
+          #- macos-latest
+          #- windows-latest
+          - ubuntu-latest
+        python-version:
+          - '3.8'
+          - '3.9'
```

Comment:
holding off on 3.10 and 3.11?

Reply:
I've tried both and got some runtime errors with `pyximport`. I'll work on that in a follow up PR